src/ocm/components/clusterWizard/ClusterPlatformIntegrationHint.tsx

```diff
@@ -27,13 +29,19 @@ export const integrationPlatformLinks: Record<SupportedPlatformType, string> = {
 export const ClusterPlatformIntegrationHint = ({
   clusterId,
   platformType,
+  openshiftVersion,
 }: ClusterPlatformIntegrationHintProps) => {
   const { isPlatformIntegrationSupported, supportedPlatformIntegration } =
     useClusterSupportedPlatforms(clusterId);
-
+  const featureSupportLevels = useFeatureSupportLevel();
+  const isNutanixFeatureSupported = featureSupportLevels.isFeatureSupported(
+    openshiftVersion || '',
+    'NUTANIX_INTEGRATION',
+  );
```

Comment:
Nit, it doesn't matter that much but...
```suggestion
  ) ?? false;
```
Just to be sure `isNutanixFeatureSupported` is always a boolean